test/test.js

```diff
@@ -9,22 +9,61 @@ const uuid = require('node-uuid');
 
 describe('nodejs-docker', () => {
 
-  it('serves traffic on 8080', (done) => {
-    runDocker('test/express', 8080, (host, callback) => {
-      setTimeout(() => {
-        request(`http://${host}:8080`, (err, res, body) => {
-          if (err) {
-            console.error(`Error requesting content: ${util.inspect(err)}`);
-            throw err;
-          }
-          assert.equal(body, 'Hello World!');
-          callback(() => {
-            done();
+  function verifyOutput(description, dockerImage, expectedOutput) {
+    it(description, (done) => {
+      runDocker(dockerImage, 8080, (host, callback) => {
+        setTimeout(() => {
+          request(`http://${host}:8080`, (err, res, body) => {
+            try {
+              if (err) {
+                console.error(`Error requesting content: ${util.inspect(err)}`);
+                throw err;
+              }
+              assert.equal(body, expectedOutput);
+            }
+            finally {
+              callback(() => {
+                done();
```

Comment:
Hmm, a simple test case replicating this behavior passes:

```javascript
describe('test', function() {
  it('foo', function(done) {
    setTimeout(function() {
      try {
        throw new Error('foo');
      } finally {
        done();
      }
    }, 5);
  });
});
```

It would be good to understand what is causing the difference but it isn't necessary for this PR if the test is functional.